src/css-select-parse5-adapter.ts

```diff
@@ -0,0 +1,94 @@
+import {Adapter as CSSSelectAdapter} from 'css-select';
+import {Attribute, Element, Node, TreeAdapter} from 'parse5';
+const defaultTreeAdapter = require('parse5/lib/tree-adapters/default');
```

Comment:
style: add new line after import section, between the exports, and between class methods.

Reply:
👍